sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -0,0 +1,123 @@
+package io.sentry.logback
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.LoggerContext
+import com.nhaarman.mockitokotlin2.check
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import io.sentry.core.HubAdapter
+import io.sentry.core.SentryLevel
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+
+class SentryAppenderTest {
+    private class Fixture {
+        val hubAdapter = mock<HubAdapter>()
+        val logger: Logger = LoggerFactory.getLogger(SentryAppenderTest::class.java)
+
+        init {
+            val loggerContext = LoggerFactory.getILoggerFactory() as LoggerContext
+
+            val appender = SentryAppender(hubAdapter)
+            appender.context = loggerContext
+
+            val rootLogger = loggerContext.getLogger(Logger.ROOT_LOGGER_NAME)
+            rootLogger.level = Level.TRACE
+            rootLogger.addAppender(appender)
+
+            appender.start()
+            loggerContext.start()
+        }
+    }
+
+    private val fixture = Fixture()
+
+    @Test
+    fun `converts message`() {
+        fixture.logger.debug("testing message conversion", 1, 2)
```

Comment:
Would be nice to have some templates like (guessing the format): 

```suggestion
        fixture.logger.debug("testing message conversion %d %d", 1, 2)
```

Then we can assert the `message` (template) and the `formatted` field